### DISCUSSION TITLE
New feature in v 1.1: adding simple push mount time to computer function

### DISCUSSION THREAD

**Discussion by fenriques** [mworion/MountWizzard4#116] [Closed Feature requests]
hi Michel,
I saw that in the new (not yet released) version of MW4 you will implement a time sync feature, at least is what it seems from the title.
Will the mount update the time on the connected pc where MW4 runs or will be the other way around (the pc updating the mount)?

To me it make more sense it's the pc updating the mount as the pc has NTP that keeps the time in sync.

Now there's the 10Micron mount sync tool for this but it only runs on Windows, so I've to start a win VM just to sync the mount time.
It would be great if MW4 could do just the same.

Ferrante

**Comment by mworion** [comment]
Hi Ferrante, 
the idea was really (as discussed in the thread) not to build a time sync but more a time takeover if you mount in synced with GPS and your mount computer might lost time due to missing power supply. The accuracy is only about 0.5s.
MW4 uses the mount time base as internal timebase for all things happening. As from python side I can't (or I not good enough) access the round tip times of the packets over all platforms well with the needed precision, It would be a bald solution so far.
Sorry to state that right now.
Michel

**Reply by mworion** [reply]
IMplemented
